src/Native/CMakeLists.txt

```diff
@@ -126,18 +126,14 @@ if(CLR_CMAKE_PLATFORM_ARCH_AMD64)
   add_definitions(-D_TARGET_AMD64_=1)
   add_definitions(-D_AMD64_)
   add_definitions(-DAMD64)
```

Comment:
We should be consistent - either have the version without underscore everywhere or nowhere.  Having it everywhere would be future proofing - it should be only needed if we ever start processing C# files with C# x-plat ifdefs via C++ preprocessor. I would vote for having it nowhere since we do not need it today.